en/pragma.tex

```diff
@@ -304,6 +304,20 @@ \subsubsection{} In casual conversation the reflexive perfective of
 \NTeri{3/8/2011}{http://naviteri.org/2011/08/new-vocabulary-clothing/}
 
 
+\subsubsection{Slang} In informal contexts some constructions that
+deviate from standard grammar may be used to serve certain purposes.
+\index{register!slang}\
+
+\subsubsubsection{}The mood infixes \N{<ei>} and \N{<äng>}, normally used in verbs,
+may be inserted into \N{<srane>}, \E{<yes>} and \N{<kehe>}, \E{<no>} instead of using
```

Comment:
I'd drop the commas before the English translations here (in the rest of the document usually \N{lì'u} \E{word} is used without a comma)

Reply:
Tsun tivam